ansible/roles/atmo-realvncserver/tasks/main.yml

```diff
@@ -123,6 +123,14 @@
   tags:
     - kill-vnc
 
+- name: Enable unencrypted connections to RealVNC
+  lineinfile:
+    dest: /etc/vnc/config.custom
+    state: present
+    backrefs: yes
+    regexp: "^SecurityTypes=RA2"
+    line: "SecurityTypes=RA2,VncAuth"
```

Comment:
What would be the best way to do this? The `GUACAMOLE_ENABLED` variable is on clank in troposphere's group_vars, so I could probably just add the same line: 
`GUACAMOLE_ENABLED: "{{ GUACAMOLE_ENABLED | default(False) }}"`
to the atmosphere_ansible group_vars